Sources/DNA/Color/Color+Button.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import Foundation
+
+public extension UIColor {
+    public class var callToActionButtonHighlightedBodyColor: UIColor {
```

Comment:
also, why not use static instead of class? :)

Reply:
I would probably use `public static let`, but the UIColor extensions in `Color.swift` used `class var` so I did it the same way for consistency.

Reply:
I think because this is how it's defined in UIKit. Might play better with ObjC but I'm not sure.

Reply:
ObjC interop might be the reason unless they actually want them to be overridable. Cause I guess thats the only other difference between class methods/properties and static methods/properties? 
At least I think I've read that runtime look up for class and static methods/properties are the same.